backend-ktor/src/main/kotlin/me/zwsmith/plugins/Models.kt

```diff
@@ -0,0 +1,22 @@
+package me.zwsmith.plugins
```

Comment:
What's the significance of the `plugins` folder?

Do these models belong in it or a more generic folder?

Reply:
i'm not sure. it's part of the ktor architectural philosophy but i'm not sure what the right way to organize stuff is. https://ktor.io/docs/server-plugins.html